src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationContext.cs

```diff
@@ -107,14 +108,35 @@ public AuthenticationContext(string authority, bool validateAuthority, TokenCach
         {
         }
 
+        /// <summary>
+        /// Constructor to create the context with the address of the authority and flag to turn address validation off.
+        /// Using this constructor, address validation can be turned off. Make sure you are aware of the security implication of not validating the address.
+        /// 
+        /// This constructor allows the HttpClient to be provided instead of having a new one created.  This can be useful when working with multiple different tenants.
```

Comment:
>This constructor allows the HttpClient to be provided instead of having a new one created.  This can be useful when working with multiple different tenants. [](start = 12, length = 156)

This should likely be in the remarks section and also we should have an aka.ms link to the wiki explaining this in more detail. Basically this allows for full control over the httpClient passed in including httpHandlers, which can be used for more control, but also for additional error handling and additional telemetry/logging.